src/components/Forms/SignInForm/SignInForm.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Link from '@material-ui/core/Link';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './style';
+
+function Copyright() {
```

Comment:
Just a matter of coding style, since you are using ES6 syntax below, you may want to be consistent here as well.